### PR TITLE
[dom-gpu] Updating production list for CLE7

### PR DESCRIPTION
**Pull request by jfavre**
Adding ParaView OSMesa

### CODE REVIEW

**Comment by lucamar** [comment]
This pull request will be `WIP` until #1102 and the other missing applications will be added in production.

File: jenkins-builds/7.0.UP00-19.03-mc [hunk 1]
```diff
@@ -58,4 +59,9 @@
  papi-wrap-1.0-CrayCCE-19.03.eb                                          --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  papi-wrap-1.0-CrayIntel-19.03.eb                                        --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  pudb-2018.1-CrayGNU-19.03-python3.eb               --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Scalasca-2.5-CrayGNU-19.03.eb                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Score-P-5.0-CrayCCE-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Score-P-5.0-CrayGNU-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Score-P-5.0-CrayGNU-19.03-cuda-10.0.eb                                  --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
```
Comment (jgphpc) [inline jenkins-builds/7.0.UP00-19.03-mc]: only in dom-gpu

File: jenkins-builds/7.0.UP00-19.03-mc [hunk 2]
```diff
@@ -58,4 +59,9 @@
  papi-wrap-1.0-CrayCCE-19.03.eb                                          --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  papi-wrap-1.0-CrayIntel-19.03.eb                                        --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
  pudb-2018.1-CrayGNU-19.03-python3.eb               --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Scalasca-2.5-CrayGNU-19.03.eb                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Score-P-5.0-CrayCCE-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
+ Score-P-5.0-CrayGNU-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
```
Comment (jgphpc) [inline jenkins-builds/7.0.UP00-19.03-mc]: add `--set-default-module`

File: jenkins-builds/7.0.UP00-19.03-gpu [hunk 3]
```diff
@@ -65,4 +67,9 @@
  papi-wrap-1.0-CrayCCE-19.03.eb                                          --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  papi-wrap-1.0-CrayIntel-19.03.eb                                        --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  pudb-2018.1-CrayGNU-19.03-python3.eb               --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
+ Scalasca-2.5-CrayGNU-19.03.eb                      --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
+ Score-P-5.0-CrayCCE-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
+ Score-P-5.0-CrayGNU-19.03.eb                                            --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
```
Comment (jgphpc) [inline jenkins-builds/7.0.UP00-19.03-gpu]: add `--set-default-module`

**Pull request merged**